docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -141,6 +141,103 @@ which groups these values were observed, as well as the difference and ratio
+^^^^^^^^^^^^^^^^^^^^
+
+This trap occurs when data scientists assume that the best solution to a problem involves
+technology, and fail to recognize other possible solutions outside of this realm.
+One area where this manifests in is within contexts in which the definition "fairness"
+changes or is dependent on a political context.
+
+Example: consider the problem of internet connectivity in rural communities. An example
+of the solutionism trap is assuming that by using data science to study internet speed in
+a given region, policies and deals can be arranged to reduce the price of internet access.
+However, if there are additional socioeconomic problems within a community, for example lack of
+education, infrastructure, information technology and health services, then an
+algorithmic solution purely focused on internet speed will fail to meaningfully address
+the needs of the community.
+
+The Ripple Effect Trap
+^^^^^^^^^^^^^^^^^^^^^^
+
+Selbst et al. [#4]_ define this as a "failure to understand how the insertion of technology into
+an existing social system changes the behaviors and embedded values of the pre-existing system".
+
+Example 1:
+
+The Formalism Trap
+^^^^^^^^^^^^^^^^^^
+
+This trap occurs when implementing an algorithmic solution failing to account the
+social dimensions associated to a problem. These dimensions include processes involved,
+whether human, political or bureaucratic, as well as cultural and time-bound social
+contexts. Such dimensions can often not be resolved via mathematical means.
+Failure occurs when steps are not taken to eradicate the limitations behind mathematical
+and statistical models.
+
+Example: 
+
+The Portability Trap
+^^^^^^^^^^^^^^^^^^^^
+
+This trap occurs when data scientists fail to understand how reusing a model or algorithm
+that is designed for one specific social context, may not necessarily apply to a different social
+context. Reusing an algorithmic solution and failing to take into account differences in
+involved social contexts can result in misleading results and potentially harmful consequences
+if the algorithm is used to determine the fate of an individual.
+
+Example: Reusing a machine learning algorithm used to screen job applications in the nursing
+industry, for job applications in the information technology sector. An intuitive yet important
+difference between both context is the difference in skills required to succeed in both industries.
+A slightly more subtle difference is the demographic differences in genders attracted to each
+context, resulting from wording in job postings, social constructs and the male-female ratio
+of successful applicants in each field.
+
+The Framing Trap
+^^^^^^^^^^^^^^^^
+
+This trap occurs when data scientists fail to consider the full picture surrounding
+a particular social context when designing and implementing a machine learning model
+in which the outcome involves enforcing decisions that will impact a person or group of people.
```

Comment:
I think we can emphasize here that the narrow framing could exclude other technological systems but particularly social actors that influence the outcome of the system, in particular the system's users.

Reply:
Solved by incorporating your suggested paragraph @hildeweerts

Reply:
Added frames to emphasize the importance of incorporating both techncological and social systems